src/99-appendices/08-coordinate-systems.md

```diff
@@ -47,53 +47,82 @@ different conventions used in specific software packages developed by different
 
 Generally, across the MEG, EEG, and iEEG modalities, the first two pieces of
 information for a coordinate system (origin and orientation) are specified in
-`<datatype>CoordinateSystem`.
+`<CoordSysType>CoordinateSystem`.
 The third piece of information for a coordinate system (units) are specified in
-`<datatype>CoordinateUnits`.
-Here, `<datatype>` can be one of `MEG`, `EEG`, or `iEEG`, depending on the
-modality that is being used.
-
-Allowed values for the `<datatype>CoordinateSystem` field come from a list of
+`<CoordSysType>CoordinateUnits`.
+Here, `<CoordSysType>` can be one of of the following,
+depending on the data that is supposed to be codumented:
```

Comment:
Should this say documented instead of codumented?

Reply:
yes, thank you

```suggestion
depending on the data that is supposed to be documented:
```